tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/UpdateCombinationStockFeatureContext.php

```diff
@@ -173,4 +173,34 @@ private function fillCommand(UpdateCombinationStockCommand $command, array $data
             $command->setAvailableDate(new DateTime($dataRows['available date']));
         }
     }
+
+    /**
+     * @Then /^all combinations of product "([^"]*)" should have the stock policy to "([^"]*)"$/
+     */
+    public function allCombinationsOfProductShouldHaveTheStockPolicyTo(string $reference, string $outOfStock)
+    {
+        $product = $this->getProductForEditing($reference);
+
+        $outOfStockInt = $this->convertOutOfStockToInt($outOfStock);
+        Assert::assertSame(
+            $product->getStockInformation()->getOutOfStockType(),
+            $outOfStockInt
+        );
+
+        $combinations = $this->getCombinationsList($reference);
+
+        foreach ($combinations->getCombinations() as $combination) {
+            $stock = new \StockAvailableCore($combination->getCombinationId());
```

Comment:
```suggestion
            $stock = new StockAvailable($combination->getCombinationId());
```

Please add a `use` statement for this class, we try to never use the FQCDN even for legacy classes that have no namespaces. And we also never use the `*Core` classes but the final ones.